File: Algorithms/Fizz_Buzz/Fizz_Buzz.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        li = []
        
        for i in range(1, n + 1):
            if (i % 5 == 0) and (i % 3 == 0):
                li.append("FizzBuzz")
            elif i % 5 == 0:
                li.append("Buzz")
            elif i % 3 == 0:
                li.append("Fizz")
            else:
                li.append(str(i))
                
        return li

File: Algorithms/Fizz_Buzz/Fizz_Buzz.cpp
class Solution {
public:
    vector<string> fizzBuzz(int n) 
    {
        vector<string> li;
        
        for(int i = 1; i <= n; i++)
        {
            if ((i % 5 == 0) && (i % 3 == 0))
                li.push_back("FizzBuzz");
            else if (i % 5 == 0)
                li.push_back("Buzz");
            else if (i % 3 == 0)
                li.push_back("Fizz");
            else
                li.push_back(to_string(i));
        }
        
        return li;
    }
};
